highfidelity/hifi
# Case 20778 - pull folder from Android

https://highfidelity.fogbugz.com/f/cases/edit/20778/Connect-nitpick-to-mobile-devices
# Description
nitpick has been enhanced to be able to pull a folder from a connected Android device
# Testing
Perform tests on both Windows and Mac.
1.  Download PR
2.  Run nitpick
3.  Create an empty working folder on the computer
4.  Create a folder with a few files on an Android device
5.  Connect Android device to computer
6.  Run nitpick
7.  Select the `Test on Device` tab
8.  Set the working folder to the folder created in Step 3
9. Click `Connect Device` and verify that the label shows the device ID (a hexadecimal string)
10.  Enter the Android folder name into the text box
11. Click `Pull folder`
12. Verify that the folder has been copied to the working folder

13.  Select the `Test on Desktop` tab
14.  Verify that the desktop full test still works.
